web/pdf_viewer.js

```diff
@@ -230,19 +230,15 @@ class PDFViewer {
       );
     }
     this.container = options.container;
-    this.#resizeObserver.observe(this.container);
-
     this.viewer = options.viewer || options.container.firstElementChild;
 
     if (
       typeof PDFJSDev === "undefined" ||
       PDFJSDev.test("!PRODUCTION || GENERIC")
     ) {
       if (
-        !(
-          this.container?.tagName.toUpperCase() === "DIV" &&
-          this.viewer?.tagName.toUpperCase() === "DIV"
-        )
+        this.container?.tagName.toUpperCase() !== "DIV" ||
```

Comment:
Unrelated to this patch, but could we remove the call to `toUpperCase` ?
According to https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName#value, it should be useless in our case.

Reply:
I suppose the question is if we're, at all, worried about a third-party user trying to use the viewer in an XML document?
In that case this *might* matter, but probably not enough to worry about here (and this code won't run in MOZCENTRAL builds thanks to the pre-processor).